examples/afl_frida/libtestinstr.c

```diff
@@ -0,0 +1,35 @@
+/*
+   american fuzzy lop++ - a trivial program to test the build
```

Comment:
Disclaimer is not accurate, I guess. Zalewski and Google had nothing to do with it?

Reply:
it is test-instr with the change that it is test(buf,len) instead of main(...) so I think it is fair